src/main/java/seedu/address/model/entitylist/ParticipantList.java

```diff
@@ -17,13 +22,16 @@
 public class ParticipantList extends EntityList {
     private static int lastUsedId = 0;
 
-    private List<Participant> participants;
+    private final ObservableList<Participant> participants = FXCollections.observableArrayList();
+    private final ObservableList<Participant> internalUnmodifiableList =
+            FXCollections.unmodifiableObservableList(participants);
+
 
     /**
      * Constructor.
      */
     public ParticipantList() {
-        this.participants = new ArrayList<>();
+
```

Comment:
Would we need to initialize a fx observable list here?

Reply:
Understand that it is auto initialized at the top of the class, but curious as to why not in the constructor

Reply:
initialized at the start, so it would be empty by default

Reply:
hmmm then can remove the empty constructor